run_model

```diff
@@ -132,3 +132,8 @@ if __name__=="__main__":
         if not args["common"].quiet: print(f'Running Delphes ({log_fname})')
         run_cmd(delphes_cmd, log_fname, args["common"].verbose, shell=True)
         if args["common"].verbose: print(f'wrote {root_fname}')
+        
+    if 'hist' in args["common"].steps:
+        root_fname = "events.root"
```

Comment:
instead of copying this value, move the first definition above outside of the `if` statement (e.g. into the part commented as `# intermediate output filenames`) so it can be shared by multiple conditional blocks (we always want a single source of truth to ensure consistency)